c/polyjuice.h

```diff
@@ -1020,8 +1020,41 @@ int handle_transfer(gw_context_t* ctx,
   return 0;
 }
 
+#pragma push_macro("errno")
+#undef errno
+int load_eth_eoa_type_hash(gw_context_t* ctx, uint8_t eoa_type_hash[GW_KEY_BYTES]) {
+  mol_seg_t rollup_config_seg;
+  rollup_config_seg.ptr = ctx->rollup_config;
+  rollup_config_seg.size = ctx->rollup_config_size;
+
+  mol_seg_t allowed_eoa_list_seg =
+      MolReader_RollupConfig_get_allowed_eoa_type_hashes(&rollup_config_seg);
+  uint32_t len = MolReader_AllowedTypeHashVec_length(&allowed_eoa_list_seg);
+  for (uint32_t i = 0; i < len; i++) {
+    mol_seg_res_t allowed_type_hash_res =
+        MolReader_AllowedTypeHashVec_get(&allowed_eoa_list_seg, i);
+
+    if (allowed_type_hash_res.errno != MOL_OK) {
```

Comment:
Maybe we function [is_errno_ok](https://github.com/nervosnetwork/godwoken-polyjuice/blob/1.3.0/c/polyjuice_utils.h#L60-L66) is a better choice here.

Reply:
Done